scripts/build-dist-cp27

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```

Comment:
these changes should be in their own PR.

also, do we still need this workflow? we get wheel building in GitHub Actions, and we have means of publishing dev builds to PyPI, we shouldn't need manual building/artifacts from CircleCI?